library/camel-kamelets-utils/src/main/java/org/apache/camel/kamelets/utils/serialization/InflightProtobufSchemaResolver.java

```diff
@@ -0,0 +1,25 @@
+package org.apache.camel.kamelets.utils.serialization;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.FormatSchema;
+import com.fasterxml.jackson.dataformat.protobuf.schema.ProtobufSchema;
+import com.fasterxml.jackson.dataformat.protobuf.schema.ProtobufSchemaLoader;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.RuntimeCamelException;
+import org.apache.camel.component.jackson.SchemaResolver;
+
+public class InflightProtobufSchemaResolver implements SchemaResolver {
+
+    @Override
+    public FormatSchema resolve(Exchange exchange) {
+        String schemaStr = (String) exchange.getProperty("schema");
```

Comment:
Should the schema property be removed also, otherwise it leaks out of the kamelet.

Reply:
Done in the yaml, to have them in one place